prow/prowjobs/GoogleCloudPlatform/gcp-guest/OWNERS

```diff
@@ -0,0 +1,4 @@
+hopkiw
+rofuentes
+iamsubratp
+wrigri
```

Comment:
This is not a valid OWNERS file.
(Please enable the `verify-owners` plugin anywhere OWNERS files are used to prevent merging invalid OWNERS files and breaking automation.)